.eslintrc.js

```diff
@@ -9,4 +9,23 @@ module.exports = {
     'plugin:prettier/recommended',
   ],
   plugins: ['prettier'],
+  overrides: [
+    {
+      files: ['**/*.ts'],
+      rules: {
+        /*
+         * The mocha plugin reports the following signature as a violation of
+         * `mocha/handle-done-callback` (misinterpreting `this` as `done`).
+         *
+         * ```ts
+         * before(async function setupApplication(this: Mocha.Context) {
+         *   this.timeout(6000);
+         *   // ...
+         * }
+         * ```
+         */
+        'mocha/handle-done-callback': 'off',
```

Comment:
I think it's better to disable this rule in `@loopback/eslint-config`, to fix the problem for all consumers, including our loopback4-example-shopping and the IBM products we are working on.

WDYT?

Reply:
I initially had the change in `eslint-config` but I think it's too invasive. The usage of `(this: Mocha.Context)` signature is probably not very common. I prefer to leave it out for now.